src/components/TodoComponents/Todo.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import TodoList from './TodoList'
-
```

Comment:
Line 2 is not needed since you're not using `<TodoList />` here, so you can just remove it.